.gitignore

```diff
@@ -347,7 +360,6 @@ coverage.opencover.xml
 .store/
 dotnet-sonarscanner
 reportgenerator
```

Comment:
I don't think we should allow these files to be included, they could contain sensitive information

Reply:
for sensitive information i'd suggest we encorage the use of dotnet's user-secrets.

I needed to add some of these files to get the tests passing.  The Testing.Tests test the configuration and require files like this.

launchsettings.json is how the program is run, i've found it helpful in the past for all devs to share the same options, happy to be challenged on this.

If your keen to add these back in, we can do so now that the files we needed for the Testing.Tests are added

Reply:
I think my concern with this is that on other projects these have been ignored and therefore Devs leave keys in them. This is then a risk for it being exposed to GitHub. Especially as it is a change from the Microsoft standard.

Reply:
re-added